olp-cpp-sdk-core/include/olp/core/client/ApiNoResult.h

```diff
@@ -29,5 +32,8 @@ namespace client {
  */
 class ApiNoResult {};
```

Comment:
Doesn't this class also need the `CORE_API` declaration?
```suggestion
class CORE_API ApiNoResult {};
```

Reply:
This is a header-only class so I guess nothing to export